telos-frontend/src/components/habit-tracker/DashboardHabitTracker.jsx

```diff
@@ -1,32 +1,323 @@
+import { useState } from 'react';
 import AddIcon from '@material-ui/icons/Add';
 import IconButton from '@material-ui/core/IconButton';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
```

Comment:
instead of importing like this, would it be nicer to import like:
import { IconButton, Box, Button, etc } from '@material-ui/core'?